src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/template.json

```diff
@@ -227,6 +228,11 @@
       "id": "set-startup-project",
       "description": "Sets the startup project in the solution",
       "actionId": "5BECCC32-4D5A-4476-A0F9-BD2E81AF0689",
+      "manualInstructions": [
+        {
+          "text": "Set the startup project in the solution"
```

Comment:
Did you verify that this **isn't** shown in VS? When I tried to set this recently to suppress this warning, VS would always show this message 😞

Reply:
I don't recall. Can it be an empty string?